active/fundraising.md

```diff
@@ -2,31 +2,60 @@
 
 ## Scope of responsibilities
 
-Coordinates fundraising efforts, particularly around corporate and major donations.
+This working group coordinates fundraising efforts, particularly around corporate and major donations.
+
+The goals of the group are:
+
+- Acquire funding to secure the financial future of Django and support future growth of Django
+- Build and strengthen relationships between Django and donor sources
+- Raise awareness of the Django Web Framework and its community with the broader world
+
+Delegated responsibilities:
+
+- Contacting, developing and implementing financial sponsorships with donor sources
+- Clarifying and elaborating on the various creative strategies that donor sources of different sizes and types can use to support Django
 
-Delegated responsibilities TBD.
 
 ## Initial membership
 
 - Chair: TBD
 - Co-Chair: TBD
 - Board Liaison (must be an active Board member; may be the same as Chair/Co-Chair): TBD
-- Other members: TBD
+- Other members:
+  - Jacob Kaplan-Moss
+  - Priya Pahwa
+  - Thibaud Colas
+  - Tom Carrick
+  - Sarah Abderemane
+  - Stephanie Goulet
+
 
 ## Future membership
 
 Direct membership: new members may self-nominate; the WG will vote (50%+1) to approve/deny new members; the WG will directly vote on new Chair/Co-Chairs.
 
-This committee is currently forming and seeking volinteers, including chair and co-chair. 
+This committee is currently forming and seeking volunteers, including chair and co-chair. 
+
+### Navigating Conflicts of Interest
+Each new member must disclose any potential conflicts of interest. These disclosures will be made transparent by means of ____. 
+
+Any person with a conflict of interest will need to recuse themselves from any votes or other decision-making related to their conflict.
+
+No more than X% of the Fundraising Working Group may contain individuals associated with the same organization
+
 
 ## Budget
 
 No allocated funds.
 
 ## Comms
 
-TBD
+- Collaborative documents
+- Private channel in DSF slack
```

Comment:
Not in love with Slack but it is already there. "Slack" should probably be capitalised.

Reply:
I don't prefer Slack either (my preference would be private Django Forum thread if possible or email). 

I don't want a discussion on the specific communication method to hold up moving the charter forward though — I imagine anything like this can be adjusted pretty easily once we meet and determine what communication methods everyone prefers (aka: I'd use Slack if it makes the most sense for the group).

Reply:
@thibaudcolas @priyapahwa @jacobian @cgl any opinion?